Core/Compatibility.cpp

```diff
@@ -111,3 +120,9 @@ void Compatibility::CheckSetting(IniFile &iniFile, const std::string &gameID, co
 		*flag |= all;
 	}
 }
+
+void Compatibility::CheckSetting(IniFile &iniFile, const std::string &gameID, const char *option, float *flag) {
+	std::string value;
+	iniFile.Get(option, gameID.c_str(), &value, "0");
+	*flag = stof(value);
```

Comment:
don't need `stof`, you can just get a float directly from the inifile without going through a string.

Reply:
`class Section` has a method to parse floats, `class IniFile` doesn't...

Reply:
hm, fair enough. Feel free to add it though!